tools/user_migration/migrate_user.py

```diff
@@ -236,18 +242,30 @@ def dumper(columns, values):
     return result
 
 
-def mark_user(databases, user):
+def mark_user(databases, user, state=MigrationState.IN_PROGRESS):
     """ mark a user in migration """
-    mysql = databases['mysql'].cursor()
     try:
-        logging.info("Marking {} as migrating...".format(user))
-        mysql.execute(
-            "INSERT INTO migration (fxa_uid, started) VALUES (%s, %s)",
-            (user, int(time.time()),)
-        )
-        databases['mysql'].commit()
-    except IntegrityError:
-        return False
+        mysql = databases['mysql'].cursor()
+        if state == MigrationState.IN_PROGRESS:
+            try:
+                logging.info("Marking {} as migrating...".format(user))
+                mysql.execute(
+                    "INSERT INTO migration (fxa_uid, started, state) VALUES (%s, %s, %s)",
+                    (user, int(time.time()), state)
+                )
+                databases['mysql'].commit()
+            except IntegrityError:
+                return False
+        if state == MigrationState.COMPLETE:
+            try:
+                logging.info("Marking {} as migrating...".format(user))
+                mysql.execute(
+                    "UPDATE migration SET state = %s WHERE fxa_uid = %s",
+                    (state, user)
+                )
+                databases['mysql'].commit()
+            except IntegrityError:
+                return False
```

Comment:
Should this be logged maybe or handled by the caller (it's ignored for `COMPLETE`)

Reply:
Heh, I actually think that this is a copy/paste thing. I don't believe that an Update would trigger the IntegrityError since it doesn't alter the key index, just sets the value. Will remove that.